code/switch.js

```diff
@@ -0,0 +1,192 @@
+// Step 1 - Welcome and introduction
+// Ordering process starts
+alert(
+  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+)
+
+// Asking for the customer name and storing it in a variable
+const customerName = prompt(
+  `Hi and welcome! First and foremost, what is you name?`
+)
+
+// Gretting the customer with an alert
+alert(
+  `Happy to see you here, ${customerName}! Now, let's get ordering!`
+)
+
+// Step 2 - Food choice
+// Customer is given three food alternatives, promting the customer to choose one number
+const foodType = prompt(
+  `What do you feel like eating today? Type a number 1-3.
+  \n1. Pizza
+  \n2. Pasta
+  \n3. Salad`
+)
+
+// Chosen food is saved in a variable and then shown to the customer via an alert, using an switch case
+let chosenFood
+let pizzaType
+let pastaType
+let saladType
+let subtypeChoice
+
+// Step 3 - Subtype choice
+// Nested switch case for checking subtype choices at the same time as main choice. 
+switch (foodType) {
+  case "1":
+    chosenFood = "Pizza"
+    alert(
+      `You've chosen ${chosenFood}!`
+    )
+    pizzaType = prompt(
+      `Delicious! What kind of ${chosenFood} do you want? Type a number.
+    \n1. Quattro Formaggio
+    \n2. Margherita
+    \n3. Swedish Kebab`
```

Comment:
Have not seen this before but it looks nice, as I understand it has the function of creating spaces between the alternatives but correct me if I'm wrong?

Reply:
Yes, exactly! I think I wrote about it in my readme, but I might have deleted it.. It makes for breaks in the visual appearance ☺️